distribution/manifest.json

```diff
@@ -8,7 +8,7 @@
 	"applications": {
 		"gecko": {
 			"id": "{a4c4eda4-fb84-4a84-b4a1-f7c1cbf2a1ad}",
-			"strict_min_version": "78.0"
+			"strict_min_version": "88.0"
```

Comment:
I could do 85. I just did 88, move it back?

Reply:
It looks like it’s 84